implementation/core/src/main/java/io/smallrye/faulttolerance/core/timeout/CompletionStageTimeout.java

```diff
@@ -34,7 +34,7 @@ private CompletionStage<V> doApply(InvocationContext<CompletionStage<V>> ctx) {
         AtomicBoolean completedWithTimeout = new AtomicBoolean(false);
         Runnable onTimeout = () -> {
             if (completedWithTimeout.compareAndSet(false, true)) {
-                LOG.debug(description + " invocation timed out");
+                LOG.debug(description + " invocation timed out (" + timeoutInMillis + "ms)");
```

Comment:
BTW missing space before 'ms' - fixed in https://github.com/smallrye/smallrye-fault-tolerance/pull/685